### ISSUE TITLE
Pass custom environment variables to rooms

### ISSUE THREAD

**Issue by Dalethium** [m1k1o/neko-rooms#130]
Looking into the source code for this program, I don't see a place to specify environment variables such as the ones below.

```
NEKO_ICELITE: 'false'
NEKO_ICESERVERS: '[{ "urls": [ "turn:10.0.0.1:3478" ], "username":"neko", "credential":"neko" }, { "urls": [ "stun:stun.nextcloud.com:3478" ] }]'
```

Is there any way we could get the ability to pass custom environment variables to each of the containers spawned by this?

**Comment by Dalethium** [comment]
I was hoping to have some added by default, more specifically.

**Comment by m1k1o** [comment]
Do you mean by default to all rooms when specifying in docker compose? Currently only available per room. But there is openapi spec and you can just create your own gui wrapper that sets those env variables for every room that you need.

**Comment by Dalethium** [comment]
Yeah, that's exactly what I mean. Do you plan on implementing something like that, or is it just outside the scope of what you want to do?

**Comment by m1k1o** [comment]
We cannot just pass all env variabled because then would be also configuration of neko rooms passed to each room. And we also cannot pass all `NEKO_` env varibles because then again `NEKO_ROOMS_` would be passed.

Maybe all `NEKO_` that do not start with `NEKO_ROOMS_` if that would not be very confusing.

**Comment by Dalethium** [comment]
That seems like a great solution, to be honest!

Or just something that is more specifically like ``NEKO_GLOBAL_`` if that's easier.
